atd-vze/src/views/Fatalities/fatalityGridTableParameters.js

```diff
@@ -68,3 +71,202 @@ export const fatalityGridTableColumns = {
     type: "String",
   },
 };
+
+export const fatalityGridTableAdvancedFilters = {
+  groupUnits: {
+    icon: "bicycle",
+    label: "Unit Type",
+    filters: [
+      {
+        id: "motor_vehicle",
+        label: "Motor Vehicle Fatality",
+        filter: {
+          where: [
+            {
+              "_or: [ { _and: [ { person: { unit: { unit_desc_id: { _eq: 1 } } } }, { _and: [ { person: { unit: { veh_body_styl_id: { _neq: 71 } } } }, { person: { unit: { veh_body_styl_id: { _neq: 90 } } } } ] } ] }, { _and: [ { primaryperson: { unit: { unit_desc_id: { _eq: 1 } } } }, { _and: [ { primaryperson: { unit: { veh_body_styl_id: { _neq: 71 } } } }, { primaryperson: { unit: { veh_body_styl_id: { _neq: 90 } } } } ] } ] } ]": null,
```

Comment:
Agreed - big shoutout to @tillyw  for working out these `where` clauses. 

I'm a little concerned that they are hard to read in this form, which makes it tricky to debug them or to grok exactly what is going on. What would you think about reformatting these with whitespace to show how the parts work together?

Here's an example of how it might be done for this first one and I'm curious what you think.

```
    filters: [
      {
        id: "motor_vehicle",
        label: "Motor Vehicle Fatality",
        filter: {
          where: [
            {
              "_or: \
                [ \
                  { _and: \
                    [ \
                      { person: { unit: { unit_desc_id: { _eq: 1 } } } }, \
                      { _and: \
                        [ \
                          { person: { unit: { veh_body_styl_id: { _neq: 71 } } } }, \
                          { person: { unit: { veh_body_styl_id: { _neq: 90 } } } } \
                        ] \
                      } \
                    ] \
                  }, \
                  { _and: \
                    [ \
                      { primaryperson: { unit: { unit_desc_id: { _eq: 1 } } } }, \
                      { _and: \
                        [ \
                          { primaryperson: { unit: { veh_body_styl_id: { _neq: 71 } } } }, \
                          { primaryperson: { unit: { veh_body_styl_id: { _neq: 90 } } } } \
                        ] \
                      } \
                    ] \
                  } \
                ] \
              " : null,
            },
          ],
        },
      },
 ... ]
```

Because these clause like structures are acting as a key (not a value) in an object, we can't use a template literal there, which would look a little cleaner without the `\`s, but I still think this is more human readable.

Reply:
Yeah this looks great, im going to implement this as well. Thanks Frank!